security_monkey/watcher.py

```diff
@@ -146,23 +145,26 @@ def changed(self):
     def slurp(self):
         """
         method to slurp configuration from AWS for whatever it is that I'm
-        interested in. This will be overriden for each technology.
+        interested in. This will be overridden for each technology.
         """
         raise NotImplementedError()
 
     def slurp_exception(self, location=None, exception=None, exception_map={}):
         """
         Logs any exceptions that happen in slurp and adds them to the exception_map
         using their location as the key.  The location is a tuple in the form:
-        (technology, account, region, item_name) that describes the object where the exception occured.
+        (technology, account, region, item_name) that describes the object where the exception occurred.
         Location can also exclude an item_name if the exception is region wide.
         """
         if location in exception_map:
             app.logger.debug("Exception map already has location {}. This should not happen.".format(location))
         exception_map[location] = exception
         app.logger.debug("Adding {} to the exceptions list. Exception was: {}".format(location, str(exception)))
 
-    def locationInExceptionMap(self, item_location, exception_map={}):
+        # Store it to the database:
+        store_exception("watcher", location, exception)
```

Comment:
The `@record_exception` decorator defaults source to "boto", but here you're setting it to "watcher".  It looks like one place `iam_role.py` sets it to "watcher", but the route53.py watcher does not set it.:   (line 41, 46, 51)

https://github.com/mikegrima/security_monkey/blob/exceptions_table/security_monkey/watchers/route53.py#L41

In `iam_role.py`, you set the source on line 74, but not on line 46:
https://github.com/mikegrima/security_monkey/blob/exceptions_table/security_monkey/watchers/iam/iam_role.py#L46